content/en/monitors/incident_management.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Incident Management
+kind: documentation
+disable_toc: true
+description: Create and manage incidents
+---
+
+{{< img src="monitors/incidents/incidents-top-1.png" alt="Incident Management"  style="width:80%;">}}
+
+## Overview
+
+Any event that may lead to a disruption in your organization’s services can be described as an incident, and it is often necessary to have a set framework for handling these events. Datadog’s Incident Management feature provides a system through which your organization can effectively identify and mitigate incidents. 
+
+Incidents live in Datadog alongside the metrics, traces, and logs you are collecting. You can view and filter incidents that are relevant to you.
+
+In the Datadog paradigm, any of the following are appropriate situations for declaring an incident:
+
+* An issue is or may be impacting customers or services
+* You do not know whether you should call an incident. Notify other people and increase severity appropriately.
+
```

Comment:
Incidents is currently in beta and available to all customers. For more information reach out to sales@datadoghq.com.